docs/self-managed/zeebe-deployment/zeebe-installation.md

```diff
@@ -16,6 +16,52 @@ Within this section you will find detailed information about:
 New to BPMN and want to learn more before moving forward? [Visit our Get Started Guides](/docs/guides/getting-started/) to learn about BPMN and orchestration.
 :::
 
+## Zeebe Gateway
```

Comment:
Now that I see it I think this should be a separate page. So that if a user navigates to `docs/next/self-managed/zeebe-deployment/zeebe-installation/`, they see links to configuration, security, operation as well als links to exporters (`docs/next/self-managed/zeebe-deployment/exporters/`) and a single gateway page.

Does that make sense and look reasonable to you?

Reply:
Cool 👍 This makes sense to me. Would you prefer to see the configuration information nested under the Zeebe Gateway tab rather than the Configuration tab as well?

Reply:
> Would you prefer to see the Gateway configuration information nested under the Zeebe Gateway tab rather than the Configuration tab?

No I think it's good that the "Zeebe Gateway" page only explains the general concept and embedded vs standalone but leaves out config and health probes.

> Where would you place the "Zeebe Gateway" section in relation to the sidebar?

I don't have a strong opinion on that, at the top is fine with me :+1: